v0/pipeline/steps/common/traceable-step-info.yaml

```diff
@@ -0,0 +1,140 @@
+title: TraceableStepInfo
+allOf:
+  - $ref: ../../common/step-spec-type.yaml
+  - type: object
+    required:
+      - auth
+      - mode
+      - target
+      - tool
+    properties:
+      advanced:
+        $ref: ../../steps/common/stoyaml-advanced-settings.yaml
+      auth:
+        $ref: ../../steps/common/stoyaml-auth.yaml
+      baseImageConnectorRefs:
+        $ref: ../../steps/ci/parameter-field-list-string.yaml
+      connectorRef:
+        type: string
+      imageTag:
+        type: string
+      imagePullPolicy:
+        oneOf:
+          - type: string
+            enum:
+              - Always
+              - Never
+              - IfNotPresent
+          - type: string
+            pattern: (<\+.+>.*)
+            minLength: 1
+      instance:
+        $ref: ../../steps/common/stoyaml-instance.yaml
+      mode:
+        oneOf:
+          - type: string
+            enum:
+              - orchestration
+              - extraction
```

Comment:
Should include ingestion too, since that would be a baseline for implementing other modes

Reply:
Ingestion is not supported for traceable